relay-dynamic-config/src/project.rs

```diff
@@ -107,6 +108,32 @@ impl Default for ProjectConfig {
     }
 }
 
+impl ProjectConfig {
+    /// Measurements are usually defined globally, but can be overriden per DNS. We therefore
+    /// first check if the project config has a value, if not we fall back on global.
+    pub fn _measurements<'a>(
+        &'a self,
+        global_config: &'a Arc<GlobalConfig>,
+    ) -> Option<&MeasurementsConfig> {
+        self.measurements
+            .as_ref()
+            .or(global_config.measurements.as_ref())
+    }
+
+    /// There's three types of tagging rules, one which is defined per dsn, one that is defined
+    /// globally for every dsn, and one that is only sometimes defined by the dsn. this method
+    /// will merge them together in the right order.
+    /// TODO(tor): Implement the maybe-override values logic.
+    pub fn _metric_conditional_tagging<'a>(
+        &'a self,
+        global_config: &'a Arc<GlobalConfig>,
+    ) -> impl Iterator<Item = &TaggingRule> {
+        self.metric_conditional_tagging
+            .iter()
+            .chain(global_config.metric_conditional_tagging.iter())
+    }
+}
```

Comment:
since we want to first check if the fetching/reading/writing logic works, we will for this PR not use this merging logic, and rather do it in a following PR

Reply:
Let's remove it then.

```suggestion
```